config/jobs/kubernetes-sigs/cluster-api-provider-vsphere/cluster-api-provider-vsphere-presubmits-main.yaml

```diff
@@ -80,83 +80,57 @@ presets:
 presubmits:
   kubernetes-sigs/cluster-api-provider-vsphere:
   - name: pull-cluster-api-provider-vsphere-apidiff-main
-    always_run: true
-    optional: true
-    decorate: true
-    path_alias: sigs.k8s.io/cluster-api-provider-vsphere
-    spec:
-      containers:
-        - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20230719-72ce785784-1.27
-          command:
-          - runner.sh
-          args:
-          - ./hack/ci-apidiff.sh
-    annotations:
-      testgrid-dashboards: vmware-cluster-api-provider-vsphere, sig-cluster-lifecycle-cluster-api-provider-vsphere
-      testgrid-tab-name: pr-apidiff
-      testgrid-alert-email: sig-cluster-lifecycle-cluster-api-vsphere-alerts@kubernetes.io
-      description: Checks for API changes in the PR
-
-  - name: pull-cluster-api-provider-vsphere-verify-markdown
+    branches:
+    - ^main$
     always_run: false
-    run_if_changed: '.*\.md$'
+    # Run if go files, scripts or configuration changed (we use the same for all jobs for simplicity).
+    run_if_changed: '^((apis|config|controllers|feature|hack|packaging|pkg|test|webhooks)/|Dockerfile|go\.mod|go\.sum|main\.go|Makefile)'
```

Comment:
Only run this if it is necessary

Reply:
Also here: already adding "webhooks" [xref](https://github.com/kubernetes/test-infra/pull/30246#discussion_r1278902177) 👍